xapian-applications/omega/docs/omegascript.rst

```diff
@@ -222,6 +222,11 @@ $fmt
 $freq{term}
 	frequency of a term
 
+$hash{HASH,TEXT}
+    unique ID for TEXT string using the hashing algorithm specified by HASH.
+    Currently, this command only supports MD5 hashing algorithm which can be
+    used by specifying HASH as "md5" (without quotes).
```

Comment:
Currently, we only have one algorithm. It best that if it works without specifying Hash Algorithm name.

We are even doing transform operation to make it all lower case. Even though we just have only one algorithm.